pywisetransfer/__init__.py

```diff
@@ -10,13 +13,25 @@ def add_resources(self):
         self.subscriptions = Subscription(client=self)
         self.users = User(client=self)
 
-    def __init__(self, api_key, environment="sandbox"):
+    def __init__(
+        self, api_key, environment="sandbox", private_key_file=None, private_key=None
+    ):
         if api_key is None:
             raise KeyError("You must provide a value for pywisetransfer.api_key")
 
         if environment not in ("sandbox", "live"):
             raise KeyError("pywisetransfer.environment must be 'sandbox' or 'live'")
 
+        if private_key_file is not None:
+            if private_key is not None:
+                raise ValueError(
+                    "Please provide only one of pywisetransfer.private_key_file or private_key"
+                )
+            key_file = open(private_key_file, "rb")
+            private_key = load_pem_private_key(key_file.read(), None)
+
         self.api_key = api_key
         self.environment = environment
-        self.add_resources()
+        self.private_key_file = private_key_file
+        self.private_key = private_key
+        self.add_resources()
```

Comment:
It looked like there was a trailing empty newline in the GitHub editor when I was resolving merge conflicts, but I guess there wasn't, and so I've removed a newline here d'oh.

Would be nice to restore the end-of-file newline to reduce the diff (I don't think I can push to a PR branch though).